runtime/src/accounts_db.rs

```diff
@@ -8246,69 +8227,6 @@ pub mod tests {
         }
     }
 
-    #[test]
-    fn test_account_balance_for_capitalization_normal() {
-        // system accounts
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(10, &Pubkey::default(), false),
-            10
-        );
-        // any random program data accounts
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(
-                10,
-                &solana_sdk::pubkey::new_rand(),
-                false,
-            ),
-            10
-        );
-    }
-
-    #[test]
-    fn test_account_balance_for_capitalization_sysvar() {
-        let normal_sysvar = solana_sdk::account::create_account_for_test(
-            &solana_sdk::slot_history::SlotHistory::default(),
-        );
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(
-                normal_sysvar.lamports,
-                &normal_sysvar.owner,
-                normal_sysvar.executable,
-            ),
-            1
-        );
-
-        // transactions can send any lamports to sysvars although this is not sensible.
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(10, &solana_sdk::sysvar::id(), false),
-            10
-        );
-    }
-
-    #[test]
-    fn test_account_balance_for_capitalization_native_program() {
-        let normal_native_program =
-            solana_sdk::native_loader::create_loadable_account_for_test("foo");
-        assert_eq!(
-            AccountsDb::account_balance_for_capitalization(
-                normal_native_program.lamports,
```

Comment:
same thing here - replace the function call with normal_native_program.lamports.